CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.24.13] - 2021-05-04
+
+### Changed
+
+- Remove monitor_agent events (#122)
```

Comment:
Would you mind extrapolating this a bit?  I don't see it documented and it's unclear to me as a user what effect this would have.

Reply:
Good point. updated